From a7efec93ae0c14372e8261c019df7326c7dc2a4c Mon Sep 17 00:00:00 2001
From: Govindraj Raja <Govindraj.Raja@imgtec.com>
Date: Mon, 20 Jul 2015 12:34:19 +0100
Subject: dts, marduk: Update mmc quirks for marduk

marduk uses micro-sd card slot and
doesnt use write protect line.

So avoid probing of the wp line.

Change-Id: I5e7b83703e63a8f3bac74b22fefcd60151b0bb17
Signed-off-by: Govindraj Raja <Govindraj.Raja@imgtec.com>
Signed-off-by: Mayank Sirotiya <Mayank.Sirotiya@imgtec.com>
---
 arch/mips/boot/dts/pistachio/pistachio_marduk.dts | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/mips/boot/dts/pistachio/pistachio_marduk.dts b/arch/mips/boot/dts/pistachio/pistachio_marduk.dts
index 143357d..e9a0759 100644
--- a/arch/mips/boot/dts/pistachio/pistachio_marduk.dts
+++ b/arch/mips/boot/dts/pistachio/pistachio_marduk.dts
@@ -99,6 +99,7 @@
 	status = "okay";
 
 	bus-width = <4>;
+	disable-wp;
 };
 
 &ir {
-- 
1.9.1

